connectionmanager.js

```diff
@@ -338,7 +338,7 @@ export default class ConnectionManager {
 
             if (!apiClient) {
 
-                apiClient = new apiClientFactory(serverUrl, appName, appVersion, deviceName, deviceId, devicePixelRatio);
+                apiClient = new apiClientFactory(appStorage, wakeOnLanFn, serverUrl, appName, appVersion, deviceName, deviceId, devicePixelRatio);
```

Comment:
I think we dropped `wakeOnLan`, cc @dkanada

Reply:
I will remove it again once @thornbill figures out a way to build this so it works with the current web repo, no worries. I will have to remove a lot more than just the wakeonlan file anyway.